### DISCUSSION TITLE
телекинез бла бла

### DISCUSSION THREAD

**Discussion by Kor-Angar** [dosinabox/g2nr_unofficial_update#460] [General]
...

**Comment by Kor-Angar** [comment]
вот это неправильно `		cond_value[2] = SPL_Cost_Scroll;` в инстанции свитка. это заклинание с инвестицией маны, а SPL_Cost_Scroll идёт только для тех что кастуется.

**Comment by Kor-Angar** [comment]


**Reply by Kor-Angar** [reply]
`SPL_Telekinesis` оказался чувствительный к регистру. в mdk Он был капсулком

**Comment by Kor-Angar** [comment]
в спелл_логик можно использовать `SPL_RECEIVEINVEST` например

```D
	func int :Spell_Logic_Telekinesis(var int manaInvested)
	{
		if	self.attribute[ATR_MANA] == 0
		{
			return SPL_SENDSTOP;
		};
		if	(manaInvested == 0)
		{
			return SPL_NEXTLEVEL;
		};
		return SPL_RECEIVEINVEST;
	};
```

переход на второй уровень заклинания необходим для включение новых анимации на некоторых заклинаниях.

**Comment by dosinabox** [comment]
А откуда я взяла анимации?

**Reply by Kor-Angar** [reply]
Не знаю. Может из Архолоса? :-) На форуме тебе ещё скидывали старый мод на телекенез может там были анимации..
